### ISSUE TITLE
Incompatible Objective-C category definitions on Xcode 12

### ISSUE THREAD

**Issue by tinder-owenthomas** [FLEXTool/FLEX#479]
I'm working on upgrading to Xcode 12 and am running into a new compiler warning relating to FLEX. I'm honestly not even sure what it means. Anyone else encountering this?

```
Incompatible Objective-C category definitions. Some category metadata may be lost.
'/Path/To/Xcode/DerivedData/App-ID/Build/Products/Debug-iphonesimulator/FLEX/FLEX.framework/FLEX(UIFont+FLEX.o)' and 
'/Path/To/My/Project/LocalFrameworks/Spotify.framework/Spotify(NSError_AudioStreaming.o) 
built with different compilers
```

**Comment by NSExceptional** [comment]
Sounds to me like Spotify might need to build their framework with a newer version of Xcode but I'm not sure, I will look into this

**Comment by tinder-owenthomas** [comment]
Yeah, wasn't sure if it was a problem with FLEX, or Spotify, or both. We have dozens of other third-party libraries and no impact on any of the others so I figured it was probably something to do with _both_ FLEX and Spotify. But not sure.

**Comment by NSExceptional** [comment]
Wait, is the Spotify framework open source? I just noticed that's a .o file

**Comment by tinder-owenthomas** [comment]
I'm pretty sure it is open source, but we're also on an older version of it. The Flex file mentioned is also a .o file though.

**Comment by NSExceptional** [comment]
I ask because if you're compiling it yourself then you should be able to compile them both "with the same compiler"

Does the error persist if you clean the build folder and do a clean build?
